<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAccessibility</class>
 <widget class="QWidget" name="DlgAccessibility">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accessibility settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="kcfg_enableAutoCompletion">
     <property name="title">
      <string>Auto completion</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="kcfg_autoCompleteMergeNumbers">
        <property name="text">
         <string>Merge numbers from different accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_autoCompleteUseAccount">
        <property name="text">
         <string>Call number using their original accounts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_pT2SBG">
     <property name="title">
      <string>Text to Speech</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="kcfg_enableVoiceFeedback">
        <property name="text">
         <string>Enable audio feedback from common actions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_enableReadLabel">
        <property name="text">
         <string>Automatically read label when an overlay is being displayed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_enableReadDetails">
        <property name="text">
         <string>Automatically read the call details when it is selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Macro</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="2">
          <widget class="RightToolButton" name="m_pRemoveTB">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="CategorizedTreeView" name="m_pMacroListTV">
           <property name="indentation">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LeftToolButton" name="m_pAddTB">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_pMacroFrm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="m_pInfoIconL">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="m_pNameL">
             <property name="accessibleDescription">
              <string comment="Macro name"/>
             </property>
             <property name="text">
              <string>Name*</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="m_pNameLE">
             <property name="placeholderText">
              <string>Identifier for this macro (toolbar text)</string>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="m_pCategoryL">
             <property name="accessibleDescription">
              <string comment="Macro Category"/>
             </property>
             <property name="text">
              <string>Category*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KComboBox" name="m_pCategoryCBB">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="m_mSequenceL">
             <property name="accessibleDescription">
              <string comment="Macro sequence"/>
             </property>
             <property name="text">
              <string>Sequence*</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="m_pSequenceLE">
             <property name="placeholderText">
              <string>The key sequence to be executed</string>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="m_pDelayL">
             <property name="accessibleDescription">
              <string comment="macro delay"/>
             </property>
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="m_pDelaySB">
             <property name="suffix">
              <string>ms</string>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="m_pDescriptionL">
             <property name="accessibleDescription">
              <string comment="Macro deskcription"/>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KLineEdit" name="m_pDescriptionLE">
             <property name="placeholderText">
              <string>A description for this macro</string>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="KShortcutWidget" name="m_pShortcuts"/>
           </item>
           <item row="5" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Shortcut</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="m_pInfoL">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Info</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KShortcutWidget</class>
   <extends>QWidget</extends>
   <header>kshortcutwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LeftToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/groupedtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>RightToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/groupedtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CategorizedTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">widgets/categorizedtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_enableAutoCompletion</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_autoCompleteMergeNumbers</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_autoCompleteUseAccount</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_enableVoiceFeedback</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_enableReadLabel</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_enableReadDetails</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgAccessibility</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changed()</slot>
 </slots>
</ui>
